engines/wage/.gitignore

```diff
@@ -0,0 +1,3 @@
+.deps
```

Comment:
Right, it was from the time when the engine lived in a separate directory. Removed.